### PR TITLE
Document the sleep tag.

### PR DESCRIPTION
**Pull request by maximelb**
## Description of the change

Adding blurb on new `lc:sleep` tag.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by rosshaleliuk — DISMISSED** [review]

File: docs/tagging.md [hunk 1]
```diff
@@ -53,3 +53,6 @@ When you tag a sensor with _lc:no_kernel_, the kernel component will not be load
 
 ### lc:debug
 When you tag a sensor with _lc:debug_, the debug version of the sensor currently assigned to the Organization will be used.
+
+### lc:sleep
+When you tag a sensor wuth _lc:sleep_, the sensor will keep its connection to the LimaCharlie Cloud, but will disable all other functionality to avoid any impact on the system.
```
Comment (rosshaleliuk) [inline docs/tagging.md:58]: Typo - wuth should be with

Comment (rosshaleliuk) [reply docs/tagging.md:58]: @maximelb

**Comment by rosshaleliuk** [comment]
@maximelb  what's the difference between this tag vs sleeper deployment? Will this sensor be counted towards the quota, just not collecting any telemetry? Can you still send commands etc? Basically, what are the limitations, reasons to use this and other stuff we'd want people to know?

**Comment by maximelb** [comment]
> @maximelb what's the difference between this tag vs sleeper deployment? Will this sensor be counted towards the quota, just not collecting any telemetry? Can you still send commands etc? Basically, what are the limitations, reasons to use this and other stuff we'd want people to know?

This tag/functionality has no impact on billing at all. It will simply prevent our advanced components from loading on the agent, resulting in a sensor that still connects to our cloud, but does not _do_ anything. This means it has no impact on the system performance-wise. No events in and out. But if the tag is removed, then our components will get loaded and normal functioning will resume as usual.

The goal here is to complete the "sleeper" feature to more accurately match expectations. Before, we would only recommend people turn off all events in Exfil control, which left the sensor to operate as usual (including performance impact). This offers a more thorough option.

**Pull request merged**